cartridges/app_storefront_custom/cartridge/controllers/Home.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+var server = require("server");
+server.extend(module.superModule);
+
+server.append("Show", function (req, res, next) {
```

Comment:
Just a note, according to the homework description, you should invoke the hook from the other cartridge here (currently you do it in the other cartridge Home.js).